src/main/java/com/eleks/auctioneer/security/SecurityConfiguration.java

```diff
@@ -0,0 +1,73 @@
+package com.eleks.auctioneer.security;
+
+import com.eleks.auctioneer.service.UserInfoUserDetailsService;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.AuthenticationProvider;
+import org.springframework.security.authentication.dao.DaoAuthenticationProvider;
+import org.springframework.security.config.annotation.method.configuration.EnableMethodSecurity;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configurers.AbstractAuthenticationFilterConfigurer;
+import org.springframework.security.config.annotation.web.configurers.AbstractHttpConfigurer;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+import org.springframework.security.web.SecurityFilterChain;
+
+@Configuration
+@EnableWebSecurity
+@EnableMethodSecurity
+public class SecurityConfiguration {
+
+    /*
+     * Hardcode first user to check if security works
+     */
+//    @Bean
+//    public UserDetailsService userDetailsService(PasswordEncoder encoder){
+//        UserDetails user = User
+//                .withUsername("Vlad")
+//                .password(encoder.encode("1234"))
+//                .build();
+//
+//        return new InMemoryUserDetailsManager(user);
+//    }
+
+
+    @Bean
+    public UserDetailsService userDetailsService(){
+        return new UserInfoUserDetailsService();
+    }
+
+    @Bean
+    public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
+        http
+                .cors(AbstractHttpConfigurer::disable)
+                .csrf(AbstractHttpConfigurer::disable)
+                .authorizeHttpRequests(auth -> auth
+                        .requestMatchers("/**").permitAll()
```

Comment:
Seems like you skip Auth for all of the endpoints. It should be skipped the "add-new-user" endpoint.